spec/03-plugins/24-rate-limiting/02-policies_spec.lua

```diff
@@ -10,12 +10,11 @@ for _, strategy in helpers.each_strategy() do
       local conf       = { route = { id = uuid() }, service = { id = uuid() } }
 
       local db
-      local dao
       local policies
 
       lazy_setup(function()
         local _
-        _, db, dao = helpers.get_db_utils(strategy)
+        _, db = helpers.get_db_utils(strategy, {})
```

Comment:
What is the reasoning behind this change? It looks superfluous.

Reply:
this means "empty list of truncations" to run, as opposed to the default, which is truncate all tables.